delta/delta_1.1.2_geometry_altitude.post.sql

```diff
@@ -0,0 +1,1401 @@
+
+  UPDATE qgep_od.wastewater_structure
+    SET
+       accessibility = NEW.ws_accessibility
+     , contract_section = NEW.ws_contract_section
+     -- , detail_geometry_geometry = NEW.detail_geometry_geometry
+     , financing = NEW.ws_financing
+     , gross_costs = NEW.ws_gross_costs
+     , identifier = NEW.ws_identifier
+     , inspection_interval = NEW.ws_inspection_interval
+     , location_name = NEW.ws_location_name
+     , records = NEW.ws_records
+     , remark = NEW.ws_remark
+     , renovation_necessity = NEW.ws_renovation_necessity
+     , replacement_value = NEW.ws_replacement_value
+     , rv_base_year = NEW.ws_rv_base_year
+     , rv_construction_type = NEW.ws_rv_construction_type
+     , status = NEW.ws_status
+     , structure_condition = NEW.ws_structure_condition
+     , subsidies = NEW.ws_subsidies
+     , year_of_construction = NEW.ws_year_of_construction
+     , year_of_replacement = NEW.ws_year_of_replacement
+     , fk_dataowner = NEW.fk_dataowner
+     , fk_provider = NEW.fk_provider
+     , last_modification = NEW.last_modification
+     , fk_owner = NEW.ws_fk_owner
+     , fk_operator = NEW.ws_fk_operator
+  WHERE obj_id = OLD.fk_wastewater_structure;
+
+
+  UPDATE qgep_od.wastewater_networkelement
+    SET
+        identifier = NEW.identifier
+      , remark = NEW.remark
+      , last_modification = NEW.last_modification
+      , fk_dataowner = NEW.fk_dataowner
+      , fk_provider = NEW.fk_provider
+      , fk_wastewater_structure = NEW.fk_wastewater_structure
+    WHERE obj_id = OLD.obj_id;
+
+  UPDATE qgep_od.reach
+    SET clear_height = NEW.clear_height
+      , coefficient_of_friction = NEW.coefficient_of_friction
+      , elevation_determination = NEW.elevation_determination
+      , horizontal_positioning = NEW.horizontal_positioning
+      , inside_coating = NEW.inside_coating
+      , length_effective = NEW.length_effective
+      , material = NEW.material
+      , progression_geometry = NEW.progression_geometry
+      , reliner_material = NEW.reliner_material
+      , reliner_nominal_size = NEW.reliner_nominal_size
+      , relining_construction = NEW.relining_construction
+      , relining_kind = NEW.relining_kind
+      , ring_stiffness = NEW.ring_stiffness
+      , slope_building_plan = NEW.slope_building_plan
+      , wall_roughness = NEW.wall_roughness
+      , fk_pipe_profile = NEW.fk_pipe_profile
+    WHERE obj_id = OLD.obj_id;
+    
+  RETURN NEW;
+END; $BODY$
+  LANGUAGE plpgsql VOLATILE;
+
+CREATE TRIGGER vw_qgep_reach_on_update
+  INSTEAD OF UPDATE
+  ON qgep_od.vw_qgep_reach
+  FOR EACH ROW
+  EXECUTE PROCEDURE qgep_od.vw_qgep_reach_on_update();
+
+-- REACH DELETE
+-- Rule: vw_qgep_reach_on_delete()
+
+CREATE OR REPLACE RULE vw_qgep_reach_on_delete AS ON DELETE TO qgep_od.vw_qgep_reach DO INSTEAD (
+  DELETE FROM qgep_od.reach WHERE obj_id = OLD.obj_id;
+);
+
+--missing: delete also connected wastewater_structure (and subclass channel or other), structure_parts, re_maintenance_events
```

Comment:
Is a TODO?

Reply:
This is as well a comment I took from the original script. It's a todo I guess, but not one of this request...